src/object.d

```diff
@@ -3241,6 +3241,269 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+private template Floating1(T)
```

Comment:
It was carried from somewhere else. Anyhow, I think all that code can be massaged with relative ease inside `_adCmpT`, pardon, `__cmp`.